openwrt-mdnsd-cve-2020-11750/README.md

```diff
@@ -0,0 +1,47 @@
+# OpenWRT mdns CVE-2020-11750
+
+Three out-of-bounds access issues were found in OpenWRT's mdns. They were reported to the OpenWRT security address on April 9 2020, and a fix ([1](https://git.openwrt.org/?p=project/mdnsd.git;a=commit;h=e74a3f9883199e9db7220d52b78e5fbdb4441ca3), [2](https://git.openwrt.org/?p=project/mdnsd.git;a=commit;h=cdac0460ba50dc45735f0be2e19a5a8efc3dafe1)) was released soon after.
+
```

Comment:
Please edit to line length <80 chars
Please add link to https://openwrt.org/advisory/2020-05-06-1